dev/import-beats/packages.go

```diff
@@ -54,10 +54,10 @@ func (pc *packageContent) addDatasets(ds []datasetContent) {
 				if v.beatType != dc.beatType {
 					pc.datasets[i].name = fmt.Sprintf("%s-%s", pc.datasets[i].name, pc.datasets[i].beatType)
 					dc.name = fmt.Sprintf("%s-%s", dc.name, dc.beatType)
+					pc.datasets = append(pc.datasets, dc)
 				} else {
-					dc.name = fmt.Sprintf("%s-%s", dc.name, dc.beatType)
+					pc.datasets[i] = dc
```

Comment:
It would be nice to log if we overwrite a dataset so at least looking at the log output we could tell how many are overwritten.

Reply:
Fixed